cms/static/js/views/settings/main.js

```diff
@@ -21,7 +21,9 @@ var DetailsView = ValidatingView.extend({
         // would love to move to a general superclass, but event hashes don't inherit in backbone :-(
         'focus :input' : "inputFocus",
         'blur :input' : "inputUnfocus",
-        'click .action-upload-image': "uploadImage"
+        'click .action-upload-image': "uploadImage",
+        'click .add-course-learning-info': "addLearningFields",
```

Comment:
Nit: it would have been cleaner to have the add buttons be part of the new views (so they control both their own addition and deletion, minimizing impact outside of those views).